modules/i3s/docs/api-reference/i3s-loader.md

```diff
@@ -4,7 +4,14 @@
   <img src="https://img.shields.io/badge/From-v2.1-blue.svg?style=flat-square" alt="From-v2.1" />
 </p>
 
-> The `I3SLoader` is experimental. Currently only support I3S `MeshPyramids` data format.
+I3SLoader supported formats:
+- `3DObject` 
```

Comment:
Suggest making a markdown table with supported Yes/No column. Perhaps add links to the I3S spec?